ideascube/migrations/0009_add_a_system_user.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from ideascube.models import User
+
+
+def add_user(*args):
+    User(serial='__system__', password='!!').save()
```

Comment:
I know by it may be some day (for example when we'll display who edited the settings), and it will be too bad to need a migration just for that at this point.